### PR TITLE
New: Nested Tabs - Change prefix for selectors [ed-7540]

### PR DESCRIPTION
**Pull request by hein-obox**

### CODE REVIEW

**Comment by arielk** [comment]
@hein-obox Please change all `tabs-v2` term to `nested-tabs` (PHP, JS, SCSS)

**Comment by hein-obox** [comment]
@arielk File names and class names updated.

**Comment by hein-obox** [comment]
> @hein-obox Please change all `tabs-v2` term to `nested-tabs` (PHP, JS, SCSS)

@arielk File names and class names updated.

File: includes/widgets/tabs.php [hunk 1]
```diff
@@ -117,7 +117,7 @@ protected function register_controls() {
 			]
 		);
 
-		$is_tabs_v2_active = Plugin::$instance->widgets_manager->get_widget_types( 'tabs-v2' );
+		$is_tabs_v2_active = Plugin::$instance->widgets_manager->get_widget_types( 'nested-tabs' );
```
Comment (arielk) [inline includes/widgets/tabs.php:120]: `is_nested_tabs_active`

Comment (hein-obox) [reply includes/widgets/tabs.php]: @arielk Updated.

**Review by arielk — DISMISSED** [review]

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 2]
```diff
@@ -184,10 +184,10 @@ protected function register_controls() {
...
 			'separator' => 'before',
 			'selectors_dictionary' => [
-				'top' => '--tabs-v2-direction: column; --tabs-v2-tabs-wrapper-direction: row; --tabs-v2-tabs-wrapper-width: initial;',
-				'bottom' => '--tabs-v2-direction: column-reverse; --tabs-v2-tabs-wrapper-direction: row; --tabs-v2-tabs-wrapper-width: initial;',
-				'end' => '--tabs-v2-direction: row-reverse; --tabs-v2-tabs-wrapper-direction: column; --tabs-v2-tabs-wrapper-width: 240px;',
-				'start' => '--tabs-v2-direction: row; --tabs-v2-tabs-wrapper-direction: column; --tabs-v2-tabs-wrapper-width: 240px;',
+				'top' => '--n-tabs-direction: column; --n-tabs-tabs-heading-direction: row; --n-tabs-tabs-heading-width: initial;',
+				'bottom' => '--n-tabs-direction: column-reverse; --n-tabs-tabs-heading-direction: row; --n-tabs-tabs-heading-width: initial;',
+				'end' => '--n-tabs-direction: row-reverse; --n-tabs-tabs-heading-direction: column; --n-tabs-tabs-heading-width: 240px;',
+				'start' => '--n-tabs-direction: row; --n-tabs-tabs-heading-direction: column; --n-tabs-tabs-heading-width: 240px;',
```
Comment (udidol) [inline modules/nested-tabs/widgets/nested-tabs.php:190]: I know this was the original variable name prefix, but `--n-tabs-tabs` looks a bit funny to me.

It looks like it is meant to target the tabs themselves inside the Nested Tabs widget, but it still looks a bit weird.
Maybe we should use just `--n-tab` for all of those instead? I'm not sure, what do you think @hein-obox ?

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 3]
```diff
@@ -216,10 +216,10 @@ protected function register_controls() {
 				],
 			],
 			'selectors_dictionary' => [
-				'start' => '--tabs-v2-tabs-wrapper-justify-content: flex-start; --tabs-v2-title-width: initial; --tabs-v2-title-height: initial; --tabs-v2-title-align-items: initial;',
-				'center' => '--tabs-v2-tabs-wrapper-justify-content: center; --tabs-v2-title-width: initial; --tabs-v2-title-height: initial; --tabs-v2-title-align-items: initial;',
-				'end' => '--tabs-v2-tabs-wrapper-justify-content: flex-end; --tabs-v2-title-width: initial; --tabs-v2-title-height: initial; --tabs-v2-title-align-items: initial;',
-				'stretch' => '--tabs-v2-tabs-wrapper-justify-content: initial; --tabs-v2-title-width: 100%; --tabs-v2-title-height: initial; --tabs-v2-title-align-items: initial;',
+				'start' => '--n-tabs-tabs-heading-justify-content: flex-start; --n-tabs-title-width: initial; --n-tabs-title-height: initial; --n-tabs-title-align-items: initial;',
```
Comment (udidol) [inline modules/nested-tabs/widgets/nested-tabs.php:219]: Same here

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+	--n-tabs-overflow-x: hidden;
+	--n-tabs-overflow-y: auto;
+	--n-tabs-background-color: transparent;
+	--n-tabs-display: flex;
+	--n-tabs-direction: column;
+	--n-tabs-gap: 10px;
+	--n-tabs-border-width: 1px;
+	--n-tabs-border-color: initial;
+
+	--n-tabs-tabs-heading-display: flex;
```
Comment (udidol) [inline modules/nested-tabs/assets/scss/frontend.scss:19]: `--n-tabs-tabs`? See next comment

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 5]
```diff
@@ -255,10 +255,10 @@ protected function register_controls() {
 				],
 			],
 			'selectors_dictionary' => [
-				'start' => '--tabs-v2-tabs-wrapper-justify-content: flex-start; --tabs-v2-title-width: initial; --tabs-v2-title-height: initial; --tabs-v2-title-align-items: initial;',
-				'center' => '--tabs-v2-tabs-wrapper-justify-content: center; --tabs-v2-title-width: initial; --tabs-v2-title-height: initial; --tabs-v2-title-align-items: initial;',
-				'end' => '--tabs-v2-tabs-wrapper-justify-content: flex-end; --tabs-v2-title-width: initial; --tabs-v2-title-height: initial; --tabs-v2-title-align-items: initial;',
-				'stretch' => '--tabs-v2-tabs-wrapper-justify-content: flex-start; --tabs-v2-title-width: initial; --tabs-v2-title-height: 100%; --tabs-v2-title-align-items: center;',
+				'start' => '--n-tabs-tabs-heading-justify-content: flex-start; --n-tabs-title-width: initial; --n-tabs-title-height: initial; --n-tabs-title-align-items: initial;',
```
Comment (udidol) [inline modules/nested-tabs/widgets/nested-tabs.php:258]: Same here

**Review by udidol — CHANGES_REQUESTED** [review]

File: modules/nested-tabs/widgets/nested-tabs.php (hunk 2 above)
Comment (hein-obox) [reply modules/nested-tabs/widgets/nested-tabs.php]: @udidol Agree. All updated.

**Review by udidol — APPROVED** [review]

**Pull request merged**